pyaedt/modeler/Circuit.py

```diff
@@ -111,12 +126,13 @@ def __init__(self, parent):
 
     @property
     def edb(self):
-        """
+        """EDB.
 
         Returns
         -------
         Edb
```

Comment:
```suggestion
        :class:`pyaedt.Edb`
```
This will be somewhat hard for you to do since you're not familiar with the API.  I'll help when possible.